src/components/GroupSuggestions/index.js

```diff
@@ -48,77 +49,91 @@ const GroupSuggestions = props => {
         </div>
       ) : (
         <div>
-          <p>
-            Sinulle ehdotetaan uusia ryhmiä päivittäin kiinnostusten mukaan.
-          </p>
-          <ButtonContainer
-            secondary
-            className="get-suggestions-again-button"
-            onClick={handleGetChannelsAgain}
-          >
-            Näytä uudestaan
-          </ButtonContainer>
+          {joinedChannels && joinedChannels.length < 6 ? (
```

Comment:
I think this should be `joinedChannels.length < 5`, since user with 5 groups still gets the suggestions and can join new channels.